src/attr/converters.py

```diff
@@ -7,6 +7,26 @@
 from ._make import NOTHING, Factory
 
 
+def and_(*converters):
+    """
+    A converter that composes multiple converters into one.
+
+    When called on a value, it runs all wrapped converters.
+
+    :param converters: Arbitrary number of converters.
+    :type converters: callables
+
+    .. versionadded:: ???
```

Comment:
How can one tell the versionadded?

Reply:
It's in `attr.__init__` without the `.dev0` suffix.